docs/undergrad.yaml

```diff
@@ -89,7 +89,7 @@ Group Theory:
     orbit: 'mul_action.orbit'
     quotient space: 'mul_action.orbit_equiv_quotient_stabilizer'
     class formula: ''
-    conjugacy classes: 'group.conjugates'
+    conjugacy classes: 'conj_classes'
```

Comment:
Can you please list the renamed declarations in the top PR post, so that they get recorded in the git log after merging?

Reply:
Sorry, I forgot to merge master. There are actually no changes to `undergrad.yaml` or either of these declarations in this PR.